src/games/strategy/engine/framework/ui/NewGameChooserModel.java

```diff
@@ -149,45 +152,33 @@ private static ZipProcessingResult processZipEntry(final URLClassLoader loader,
    * Open up a confirmation dialog, if user says yes, delete the map specified by
    * parameter, then show confirmation of deletion.
    */
-  private static void confirmWithUserAndThenDeleteCorruptZipFile(final File map, Optional<String> details) {
```

Comment:
What is this change for? Hard to follow as the whole method is pretty much removed/added in the diff. Not sure if this is a bad merge or not?